ecomerce-app/src/pages/productslist/Productslist.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Productslist.css";
+import { ProductCard } from "../../component/Productcard";
+import { useProduct } from "../../Context/ProductContext/Prodctcard-context";
+import { ProductFilter } from "../../component/FilterSidebar/Filter";
+import { useFilter } from "../../Context/FilterContext/Filter-context";
+
+import { SortProduct } from "../../Utils/sortProduct";
+import { ProductcategoryFilter } from "../../Utils/categoryFilter";
+import { RatingFilter } from "../../Utils/ratingFilter";
+import { Pricefilter } from "../../Utils/pricefilter";
```

Comment:
you can create index file and all utils export then add utils function using one line

Reply:
thank you for suggestion i will  do